src/locale/locales/fr/messages.po

```diff
@@ -3567,11 +3410,11 @@ msgstr "Change le thème de couleur en fonction du paramètre système"
 
 #: src/view/screens/Settings/index.tsx:514
 msgid "Set dark theme to the dark theme"
-msgstr ""
+msgstr "Définir le thème sombre sur le thème sombre"
```

Comment:
The sentences is weird in English, but we can try not to make it even weirder in French by shuffling it up a little.

```suggestion
msgstr "Choisir le thème le plus sombre comme thème sombre"
```